Bio/Seq.py

```diff
@@ -90,6 +90,196 @@ def __len__(self):
     def __getitem__(self, key):
         pass
 
+    def __bytes__(self):
+        return self[:]
```

Comment:
Should the docstring suggest subclassing ``__bytes__`` too if needed?

Reply:
The `_BioSQLSequenceData` class in BioSQL indeed was subclassing `__bytes__`, but it is not actually needed if `__getitem__` is implemented appropriately. There may still be some cases where `__bytes__` needs to be subclassed, but by default I would not encourage it if it is not needed.

Reply:
Sorry, I meant to ask if the documentation should talk about re-implementing the ``__bytes__`` method? It looks like this is covered by your description of what ``[:]`` should do.